packages/themes/src/presets/bluePlanet.css

```diff
@@ -77,7 +89,7 @@
 .show-api-client-button svg {
   color: black !important;
 }
-.download-cta,
+.references-rendered .download-cta .download-button,
 .references-rendered .markdown a {
   text-decoration: underline !important;
```

Comment:
Another important we could maybe remove

Reply:
There's nothing more annoying than adding important for these, let's properly fix them in the coming days / weeks 😓